pallets/portfolio/src/lib.rs

```diff
@@ -130,6 +141,14 @@ decl_event! {
         /// * origin DID
         /// * vector of number-name pairs
         UserPortfolios(IdentityId, Vec<(PortfolioNumber, PortfolioName)>),
+        /// Custody of a portfolio has been given to a different identity
+        ///
+        /// # Parameters
+        /// * origin DID
+        /// * portfolio number
+        /// * portfolio owner did
+        /// * portfolio custodian did
+        PortfolioCustodied(IdentityId, PortfolioNumber, IdentityId, IdentityId),
```

Comment:
"Custodied" doesn't seem to be an English word ;)
How about `PortfolioCustodianChanged`?